index.md

```diff
@@ -29,7 +29,7 @@ This code is not exhaustive or complete. It serves to distill our common underst
 
 ### Diversity Statement
 
-We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is equal. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
+We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is worthy of respect. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
```

Comment:
Ok, since you asked, I'll chime: The intent is clear. The CoC declares that the maintainers of an project that adopts this code, as the people responsible for their successful community in their project, will 1. encourage that community members treat others properly and 2. will take serious a report if someone feels they are not being treated properly. It's that simple.

We did not feel the need to engage in the semantics questions "_are_ people equal" or "are people _treated_ equally" or "are people treated _fairly_" since that defocuses the issue we were looking to address. We are not trying to elaborate on what people are, or what kind of mistreatment is greater than or equal to others. This is a template that we thought could be helpful. If you prefer to use fair, equal, divine, you can modify it if you wish. If you think the template can be improved, please suggest it in a pull request (as I did).

I suggested "worthy of respect" since I think it's clearer that "equal." But I do not think that using the more common term "equal" is a problem. I've chosen to use my suggested text in my instance of the OCoC and I look forward to seeing this text evolve (in the open source way) to get improvements from the upstream.